src/css/atoms/button.css

```diff
@@ -99,3 +103,26 @@
     text-decoration: none;
   }
 }
+
+.button-hollow:not(.button-neutral) {
+  background: transparent;
+  color: $color-default;
+
+  &:hover {
+    background: $color-default;
+    color: $color-neutral;
+  }
+
+  @each $color in primary, secondary, danger {
+
+    &.button-$(color) {
```

Comment:
I understand your point. I would say that I prefer this way because we can benefit the styles from the pure `button` and avoid repetitions.